data/mods/Aftershock/player/bionics.json

```diff
@@ -63,6 +63,86 @@
+    "upgraded_bionic": "bio_forcefield_bash_cut_weak",
+    "available_upgrades": [ "bio_forcefield_bash_cut_heavy" ],
+    "act_cost": "5 kJ",
+    "react_cost": "5 kJ",
+    "time": 1,
+    "enchantments": [ "forcefield_bash_cut_medium" ],
+    "flags": [ "BIONIC_TOGGLED" ]
+  },
+  {
+    "id": "bio_forcefield_bash_cut_heavy",
+    "type": "bionic",
+    "name": { "str": "Debt Collection Agent Personal GravyField" },
+    "description": "This class of personal gravity manipulator often sees action on both sides of the vicious ship-to-ship boarding fights that are common with both repossession and piracy.  It remains practically impenetrable to melee weapons that lack sharp piercing points.",
+    "occupied_bodyparts": [ [ "torso", 14 ] ],
+    "upgraded_bionic": "bio_forcefield_bash_cut_medium",
+    "act_cost": "10 kJ",
+    "react_cost": "10 kJ",
+    "time": 1,
+    "enchantments": [ "forcefield_bash_cut_heavy" ],
+    "flags": [ "BIONIC_TOGGLED" ]
+  },
+  {
+    "id": "bio_forcefield_ballistic_weak",
+    "type": "bionic",
+    "name": { "str": "Point Energy Conversion Field" },
+    "description": "The PEC, as it is commonly known, works by converting kinetic energy into forms that are less damaging to the wearer, such as light.  Since this can be problematic, various types of shielding have been developed that do different emissions.  However, the ability to stop ballistic and piercing damage in its tracks often makes up for any inconvenience these emissions can cause.",
+    "occupied_bodyparts": [ [ "torso", 10 ] ],
+    "available_upgrades": [ "bio_forcefield_ballistic_medium", "bio_forcefield_ballistic_heavy" ],
+    "act_cost": "1 kJ",
+    "react_cost": "1 kJ",
+    "time": 1,
+    "enchantments": [ "forcefield_ballistic_piercing_weak" ],
+    "flags": [ "BIONIC_TOGGLED" ]
+  },
+  {
+    "id": "bio_forcefield_ballistic_medium",
+    "type": "bionic",
+    "name": { "str": "Holtzman Personal Shield Generator" },
+    "description": "Personal shielding against bullets and sharp-pointed weapons comes in a variety of types.  This version is known for drastically increasing an individual soldier's lifespan while often decreasing that of anyone - friend or foe - in close proximity.",
+    "occupied_bodyparts": [ [ "torso", 12 ] ],
+    "upgraded_bionic": "bio_forcefield_ballistic_weak",
+    "available_upgrades": [ "bio_forcefield_ballistic_heavy" ],
+    "act_cost": "5 kJ",
+    "react_cost": "5 kJ",
+    "time": 1,
+    "enchantments": [ "forcefield_ballistic_piercing_medium" ],
+    "flags": [ "BIONIC_TOGGLED" ]
+  },
+  {
+    "id": "bio_forcefield_ballistic_heavy",
+    "type": "bionic",
+    "name": { "str": "Illegally modified Shield Generator" },
+    "description": "Attention!  This compact bionic module has been altered beyond safe baselines and the warranty has been violated.  Gibson Gear takes no liability for any injuries or death that may occur from these modifications.  Attention!…",
```

Comment:
Would be good to add how much armor these increase in the description, or to better yet modify the UI so these ivalues are automatically shown like for normal armor (but these would be much more complex probably).

Goes for all forcefields really.